dds/DCPS/RTPS/TypeLookup.cpp

```diff
@@ -461,13 +472,13 @@ void serialized_size(const Encoding& encoding, size_t& size,
   const XTypes::TypeLookup_Call& uni)
 {
   serialized_size_delimiter(encoding, size);
-  primitive_serialized_size(encoding, size, uni.kind);
-  switch (uni.kind) {
+  primitive_serialized_size(encoding, size, uni._d());
+  switch (uni._d()) {
   case 25318099:
```

Comment:
I know it was already like this, but can you replace 25318099 with `TypeLookup_getTypes_HashId` and 95091505 with `TypeLookup_getDependencies_HashId`?

Reply:
Done.